parser/parser.go

```diff
@@ -358,7 +364,8 @@ func (p *Parser) curPrecedence() int {
 
 // var
 func (p *Parser) parseIdentifier() ast.Expression {
-	return &ast.Identifier{Token: p.curToken, Value: p.curToken.Literal}
+	id := &ast.Identifier{Token: p.curToken, Value: p.curToken.Literal}
+	return id
```

Comment:
why 2 lines here?

Reply:
Fixed this. I may have been reacting to a complaint from my aggressive VScode go-lint extension. I love VScode, its a great IDE, but sometimes more of a nanny :-)). 

In particular it complains about unexported external functions that don't have canonical comments preceding the func def. See https://blog.golang.org/godoc-documenting-go-code. But I only did this for new funcs that I created, so there are still a few left in the code. Priorities ... Priorities...